scripts/userprefs.js

```diff
@@ -0,0 +1,14 @@
+window.addEventListener("DOMContentLoaded", () => {
+
+    let creditNameSelector = document.getElementById('credit_name');
+    let creditOtherInput = document.getElementById('credit_other');
+
+    function maybeShowOther() {
+        creditOtherInput.hidden = (creditNameSelector.value != 'other');
```

Comment:
*minor* unnecessary parentheses. Also I think eslint may complain or warn about not using `!==` over `!=`.

Reply:
I think the parentheses make it more readable. probably better to always use !==.